src/button.tsx

```diff
@@ -97,6 +110,9 @@ const renderButtonContent = ({
     setShouldRenderSpinner: (value: boolean) => void;
     renderText: (text: React.ReactNode) => React.ReactNode;
     textContentStyle?: string;
+    iconLeft?: React.ReactElement<IconProps>;
+    iconRight?: React.ReactElement<IconProps>;
```

Comment:
should we call it `endIcon` `startIcon` to keep parity with input fields?
https://github.com/Telefonica/mistica-web/blob/0cf6455360d0f3a79e21c12101d7728c7e858716/src/text-field.tsx#L28

Reply:
fixed